README.md

```diff
@@ -61,9 +52,9 @@ The `collections-contrib` artifact provides additional operations on the collect
 ### Migrating from the standard collections to the strawman
 
 There is an [entry in the FAQ](https://github.com/scala/collection-strawman/wiki/FAQ#what-are-the-breaking-changes)
-that aims to list all breaking changes.
+that aims to list all the breaking changes.
 
-A tool is being developed to automatically migrate code that uses the standard
+A tool is currently developed to automatically migrate code that uses the standard
```

Comment:
"is currently developed" is not clear English, I'm afraid — it's not a phrase a native speaker would use, and it leaves the reader uncertain of the intended meaning — does the tool exist or not?

if it's not ready yet, "is being developed" was right (and something like "is under development" would be more natural). if it is ready now, suggest "is now available"

Reply:
Thanks, I’m putting “is being developed” back!

Reply:
(okay, note that "is currently being developed" would also be idiomatic and fine, if you like the additional emphasis on "currently")